stable/kong/templates/_helpers.tpl

```diff
@@ -177,6 +181,10 @@ the extra slash.
 {{- end -}}
 
 {{- define "kong.volumeMounts" -}}
+- name: {{ template "kong.fullname" . }}-prefix-dir
+  mountPath: /kong_prefix/
+- name: {{ template "kong.fullname" . }}-tmp
+  mountPath: /tmp
```

Comment:
Why is this needed? :thinking:

Reply:
NGINX creates some files in /tmp briefly during startup, and I don't believe we have control to have it use another directory.